src/pages/{StoreProduct.slug}/p.tsx

```diff
@@ -14,6 +14,8 @@ import type {
 } from '@generated/graphql'
 import ProductShelf from 'src/components/sections/ProductShelf'
 
+import '../../styles/pages/index.scss'
```

Comment:
the same comment regarding the PLP applies here.

Reply:
It's going to be addressed in [this other PR](https://github.com/vtex-sites/base.store/pull/292).